### ISSUE TITLE
We need code for saccades

### ISSUE THREAD

**Issue by grero** [grero/DataProcessingTools#24]
Currently, we have no code for dealing with saccades.

**Comment by shihchengyen** [comment]
One of the interns was able to use pyedfread to extract Eyelink events from the EDF files. She made an eyelink class in PyHippocampus with the code.

**Comment by grero** [comment]
Oh cool! I'll try to incorporate that here as well.

**Comment by shihchengyen** [comment]
I think she is still working on using your DPT and PanGUI classes to create plotting functions for the class.

**Comment by shihchengyen** [comment]
One of the other interns was able to use your framework to create plots for another class that reads in Unity output files.

**Comment by grero** [comment]
That's great news! As you guys discovery limitations in what PanGUI can do, don't hesitate to file issues, and we'll have them sorted out. Right now, it's a bit barebones.

**Comment by shihchengyen** [comment]
Yes, we were discussing how to implement the PlotOptions functionality as well as re-creating objects after loading the HDF5 files. Have you thought about these issues?
